packages/core/src/modules/connections/ConnectionsApi.ts

```diff
@@ -44,6 +43,7 @@ export class ConnectionsApi {
   private outOfBandService: OutOfBandService
   private messageSender: MessageSender
   private trustPingService: TrustPingService
```

Comment:
I think TrustPingService and this variable can be renamed to V1TrustPingService to make it clear its version

Reply:
Agree.